docs/source/models.rst

```diff
@@ -1672,19 +1672,23 @@ The ``mlflow models`` CLI commands provide an optional ``--env-manager`` argumen
 
 .. _azureml_deployment:
 
-Deploy a ``python_function`` model on Microsoft Azure ML
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Deploy a ``python_function`` model on Azure Machine Learning
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The MLflow plugin `azureml-mlflow <https://pypi.org/project/azureml-mlflow/>`_ can deploy models to Azure ML, either to Azure Kubernetes Service (AKS) or Azure Container Instances (ACI) for real-time serving.
+The MLflow plugin `azureml-mlflow <https://pypi.org/project/azureml-mlflow/>`_ can deploy models to Azure ML for real-time inference with Azure Kubernetes Service (AKS), Azure Container Instances (ACI) and Azure Machine Learning Managed Inference.
+
+.. note:: 
+  Azure Machine Learning supports the deployment of MLflow models for batch-inference. However, this deployment target is not supported using the `azureml-mlflow <https://pypi.org/project/azureml-mlflow/>`_. Please use any of the `other supported deployment methods <https://docs.microsoft.com/en-us/azure/machine-learning/how-to-deploy-mlflow-models>`_.
 
 The resulting deployment accepts the following data formats as input:
 
 * JSON-serialized pandas DataFrames in the ``split`` orientation. For example, ``data = pandas_df.to_json(orient='split')``. This format is specified using a ``Content-Type`` request header value of ``application/json``.
+* Tensor input format as JSON-serialized lists (for tensors) and dictionary of lists (for named tensors).
 
-.. warning::
-    The ``TensorSpec`` input format is not fully supported for deployments on Azure Machine Learning at the moment. Be aware that many ``autolog()`` implementations may use ``TensorSpec`` for model's signatures when logging models and hence those deployments will fail in Azure ML.
-
-Deployments can be generated using both the Python API or MLflow CLI. In both cases, a ``JSON`` configuration file can be indicated with the details of the deployment you want to achieve. If not indicated, then a default deployment is done using Azure Container Instances (ACI) and a minimal configuration. The full specification of this configuration file can be checked at `Deployment configuration schema <https://docs.microsoft.com/en-us/azure/machine-learning/reference-azure-machine-learning-cli#deployment-configuration-schema>`_. Also, you will also need the Azure ML MLflow Tracking URI of your particular Azure ML Workspace where you want to deploy your model. You can obtain this URI in several ways:
```

Comment:
Is the following no longer accurate / relevant?

```
The full specification of this configuration file can be checked at `Deployment configuration schema <https://docs.microsoft.com/en-us/azure/machine-learning/reference-azure-machine-learning-cli#deployment-configuration-schema>`_ 
```

Reply:
It was moved to another section. The JSON specification depends on the target used. AKS and ACI uses the same specification but Managed Endpoints uses a different one as it's a compute belonging to our V2 platform.